CHANGELOG.md

```diff
@@ -3,7 +3,8 @@
 CHANGES:
 * `global.enabled` now works as documented, that is, setting `global.enabled` to false will disable everything, with individual components able to be turned on individually [GH-703](https://github.com/hashicorp/vault-helm/pull/703)
 * Default value of `-` used for injector and server to indicate that they follow `global.enabled`. [GH-703](https://github.com/hashicorp/vault-helm/pull/703)
-* Vault default image to 1.9.3
+* Vault image default to 1.9.4
```

Comment:
Would you mind changing the Vault version to 1.10.0 throughout? That'll be what we want the default at for the next release.